ccd-definition/CaseTypeTab.json

```diff
@@ -380,5 +370,25 @@
     "TabDisplayOrder": 7,
     "CaseFieldID": "c2DocumentBundle",
     "TabFieldDisplayOrder": 15
+  },
+  {
```

Comment:
why tab display order 8?

Reply:
I was waiting on the correct numbering from Amit but discussed with you and changed back